Add CoinOne, Coinspot, Cryptopia and DSX

spazvt opened the pull request:
add 'CoinOne', 'Coinspot', 'Cryptopia', and 'DSX' crypto currency
exchanges from fork
https://github.com/acceptbitcoincash/acceptbitcoincash

spazvt committed:
add CoinOne, Coinspot, Cryptopia, DSX exchanges

add 'CoinOne', 'Coinspot', 'Cryptopia', and 'DSX' crypto currency
exchanges from fork
https://github.com/acceptbitcoincash/acceptbitcoincash

mxxcon reviewed (commented):
perhaps use their official image files?
https://s3.ap-northeast-2.amazonaws.com/coinone-cloudflare-landing/icon/favicon-32x32.png
and
https://dsx.uk/assets/img/favicons/favicon-32x32.png

spazvt committed:
update Coinone, Cryptopia, and DSX favicons

Update Coinone, Cryptopia, and DSX favicons from these links:
https://s3.ap-northeast-2.amazonaws.com/coinone-cloudflare-landing/icon/favicon-32x32.png
https://pbs.twimg.com/profile_images/998432158212476929/9BWSvrll_400x400.jpg
https://dsx.uk/assets/img/favicons/favicon-32x32.png
